### PR TITLE
tickets/DM-40799: Update to giftless, and update technote infrastructure

### PR DESCRIPTION
**Pull request by athornton**
Update to giftless, and update technote infrastructure

### CODE REVIEW

**Comment by athornton** [comment]
Still needs remaining authors added.

File: index.rst [hunk 1]
```diff
@@ -1,3 +1,14 @@
+:tocdepth: 1
+
+.. sectnum::
```
Comment (jonathansick) [inline index.rst]: Delete section numbering

File: index.rst [hunk 2]
```diff
@@ -1,3 +1,14 @@
+:tocdepth: 1
```
Comment (jonathansick) [inline index.rst]: Delete tocdepth

File: index.rst [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+
+.. sectnum::
+
```
Comment (jonathansick) [inline index.rst:7]: Add the title as a top-level section

**Comment by athornton** [comment]
Obv. we don't merge until we go live with the new service, and that, this merge, and the change to the Developer Guide all happen at once.

**Pull request merged**